CMakeLists.txt

```diff
@@ -227,7 +227,7 @@ install(
 )
 
 # Headers are all combined into one common 'include' dir when MOOS is installed
-set(MOOS_INCLUDE_DIRS "include")
+set(MOOS_INCLUDE_DIRS "\${CMAKE_CURRENT_LIST_DIR}/../../../include")
```

Comment:
Ah, there's a sneaky backslash to escape the `$` character, so in the configured `MOOSConfig.cmake` file you get:
`set(MOOS_INCLUDE_DIRS "${CMAKE_CURRENT_LIST_DIR}/../../../include")`
which is then expanded when you call `find_package(MOOS)`. So it ought to be relocatable.

I can't think of a good way of using `file(RELATIVE_PATH ...)` inside `MOOSConfig.cmake`, because at that point most of the context is unavailable. You just have to know that `MOOSConfig.cmake` lives inside `<INSTALL_PREFIX>/lib/cmake/MOOS`. We could use it inside the top level `CMakeLists.txt`, but I think it would result in the same horrid relative path appearing in the configured `MOOSConfig.cmake`? Suggestions welcome!